File: Python/Arrays.py
# Find the Duplicate numbers in an given array
# make a set 
# if all the element in set == elements in array true else false

from ast import List


def duplicatearrays(nums: List[int]) -> bool:
    unique = {}
    for num in nums:
        if num in unique:
            return True
        unique.add(num)
    return False

File: Java/Arrays.java
// Find the Duplicate numbers in an given array
/*  make a hash map with HashSet
 *  if duplicates present return true else false
 */
import java.util.HashSet;

class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet<Integer> ans = new HashSet<>();
        for (int num : nums) {
            if (ans.contains(num))
                return true;
            ans.add(num);
        }
        return false;
    }
}